apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -220,4 +265,58 @@ export const folderRouter = router({
         return pick(result, defaultFolderSelect)
       },
     ),
+
+  getIndexpage: protectedProcedure
+    .input(getIndexpageSchema)
+    .query(async ({ input: { resourceId, siteId } }) => {
+      // TODO: add permissions checking
```

Comment:
issue: I think should add this perms check before merge? A bit scary to skip cos Maps team had an incident due to missing perms check. I think should be quite similar to the perms for editing normal pages

Reply:
no worries, i'll add it in hehe